.github/workflows/github_pages_preview_fallback.yml

```diff
@@ -33,5 +33,7 @@ jobs:
   # https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks
   deploy:
     runs-on: ubuntu-latest
+    permissions:
```

Comment:
Is this permission needed? I would assume this workflow needs no additional permissions.

Reply:
This is meant to reduce permissions to just writing the check, since it's meant to be a no-op.